pkg/rotator/rotator.go

```diff
@@ -521,13 +568,53 @@ func (r *ReconcileVWH) Reconcile(request reconcile.Request) (reconcile.Result, e
 		if err := r.client.Update(r.ctx, vwh); err != nil {
 			return reconcile.Result{Requeue: true}, err
 		}
+
+		// Ensure certs on CRDs.
+		if result, err := r.ensureCRDCerts(artifacts.CertPEM); err != nil {
```

Comment:
"CRD" is a misnomer here, we are ensuring CRD conversion webhooks have certs.

Reply:
Updated to `ensureCRDConvWHCerts`.